maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/report/DefaultReporterFactory.java

```diff
@@ -181,6 +183,30 @@ public void runStarting()
             log( "-------------------------------------------------------" );
             log( " T E S T S" );
             log( "-------------------------------------------------------" );
+            displayRandomization();
+        }
+    }
+
+    private boolean isRandomized()
+    {
+        return reportConfiguration.getRunOrderParameters() != null
+                && reportConfiguration.getRunOrderParameters().isRandomized();
+    }
+
+    private void displayRandomization()
+    {
+        if ( isRandomized() )
+        {
+            final Randomizer randomizer = reportConfiguration.getRunOrderParameters().getRandomizer();
+            final String pluginName = reportConfiguration.getPluginName();
+
+            log( "" );
+            log( String.format(
```

Comment:
pls use debug log here
we use to log config params like this one on debug level
users do not like to see too much logs

Reply:
This is crucial place. Without showing the seed to user, by default, this PR have no sense. It must be presented to user be able to reproduce the same exact run.